Signal/src/AppDelegate.m

```diff
@@ -333,10 +334,39 @@ - (void)application:(UIApplication *)application
 - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray * _Nullable))restorationHandler
 {
     if ([userActivity.activityType isEqualToString:@"INStartVideoCallIntent"]) {
+        DDLogInfo(@"%@ got start video call intent", self.tag);
```

Comment:
Actually there are two buttons in the contact app. One for signal audio, or one for signal video. The audio sends the INStartAudioCallIntent, the video sends INStartVideoCallIntent.

Somewhat confusingly, as you've observed, the INStartVideoCallIntent you receive when initiating a video call from contacts is the same intent the app receives when "upgrading" an existing audio call to video.